superset/assets/src/components/FilterableTable/FilterableTable.jsx

```diff
@@ -26,13 +26,9 @@ import {
   SortDirection,
   SortIndicator,
 } from 'react-virtualized';
-import { getTextDimension } from '@superset-ui/dimension';
 import TooltipWrapper from '../TooltipWrapper';
 
-function getTextWidth(text, font = '12px Roboto') {
-  return getTextDimension({ text, style: { font } }).width;
-}
-
+const CHARACTER_WIDTH = 8;
```

Comment:
@etr2460, can you test your modifications with Unicode data? You can try a query like this:

```sql
SELECT 'ｱｲｳ1234', 'ＱＲＳ12', '台北1234';
```

Reply:
tested
<img width="303" alt="Screen Shot 2019-06-12 at 5 19 14 PM" src="https://user-images.githubusercontent.com/7409244/59394879-3c3ea680-8d36-11e9-9dfb-3813d84dfe33.png">

Reply:
My concern is that strings with the same number of characters will have different widths when rendering in monospace. For example, these 3 strings all have 3 characters:

```
'ｱｲｳ',
'123',
'ＱＲＳ'
```

Note that they're rendered with different widths, since some characters are half-width and others are double.

Reply:
```python
>>> a = ['123', 'ｱｲｳ', 'ＱＲＳ']
>>> [len(i) for i in a]
[3, 3, 3]
```

Reply:
Ah, I see your point here. It is possible to get the columns to overflow with enough double wide characters, but in that case you can still hover over the column to see the full results, or copy/paste to the same effect.